engine/src/main/java/org/terasology/engine/rendering/dag/AbstractNode.java

```diff
@@ -547,8 +549,8 @@ public void dispose() {
      */
     protected void addDesiredStateChange(StateChange stateChange) {
         if (stateChange.isTheDefaultInstance()) {
-            logger.error("Attempted to add default state change {} to the set of desired state changes. (Node: {})",
-                    stateChange.getClass().getSimpleName(), this);
+            logger.atError().addArgument(() -> stateChange.getClass().getSimpleName()).addArgument(this).
+                    log("Attempted to add default state change {} to the set of desired state changes. (Node: {})");
```

Comment:
error-level log so unlikely that method called "in vain"
```suggestion
            logger.error("Attempted to add default state change {} to the set of desired state changes. (Node: {})",
                    stateChange.getClass().getSimpleName(), this); //NOPMD
```